instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -60,12 +60,15 @@ func Middleware(service string, opts ...Option) gin.HandlerFunc {
 			c.Request = c.Request.WithContext(savedCtx)
 		}()
 		ctx := cfg.Propagators.Extract(savedCtx, propagation.HeaderCarrier(c.Request.Header))
-		opts := []oteltrace.SpanStartOption{
-			oteltrace.WithAttributes(semconv.NetAttributesFromHTTPRequest("tcp", c.Request)...),
-			oteltrace.WithAttributes(semconv.EndUserAttributesFromHTTPRequest(c.Request)...),
-			oteltrace.WithAttributes(semconv.HTTPServerAttributesFromHTTPRequest(service, c.FullPath(), c.Request)...),
-			oteltrace.WithSpanKind(oteltrace.SpanKindServer),
-		}
+		opts := cfg.SpanStartOptions
+		opts = append(opts,
```

Comment:
No need to set a variable with `cfg.SpanStartOptions`. You can do directly `opts = append(cfg.SpanStartOptions, ...`

Reply:
fixed it